node/config/def.go

```diff
@@ -66,6 +66,20 @@ type DealmakingConfig struct {
 
 	Filter          string
 	RetrievalFilter string
+
+	// If UseDynamicRetrievalPricing is set to false(default), we will use the DefaultRetrievalPricingParams
+	// for pricing retrieval deals, otherwise we will use the DynamicRetrievalPricingFunc configured here.
+	UseDynamicRetrievalPricing  bool
+	DynamicRetrievalPricingFunc string
```

Comment:
Suggestion: if DynamicRetrievalPricingFunc is the empty string then we use the DefaultRetrievalPricingParams.
That way we only need one variable.

Reply:
This is no longer needed after the config refactor mentioned above.